examples/open_loop_control_fetch/run.py

```diff
@@ -75,11 +74,6 @@
 
     if args.benchmark:
         start_time_seconds = time.time()
-    else:
```

Comment:
- leave this else statement here, but include a for-loop over rendering passes, for consistency with generate_images.py (line 109)
- this is slightly cleaner than doing the operation repeatedly, because conceptually this should be a one-time operation

Reply:
I don't think we are doing `shutil.rmtree(args.image_dir, ignore_errors=True) in our generate image dataset example, which is why I removed it from here. Let me add it our generate image dataset example as well.